packages/Algernon.package/ALGExpressionType.class/class/equationPattern.st

```diff
@@ -1,4 +1,4 @@
 icon
 equationPattern
 
-	^ '( *[+\-]? *\d+)( *[+\-*\/^]? *\d+)* *'
\ No newline at end of file
+	^ '( *[+\-]? *\d+)( *[+\-*\/]? *\d+)* *'
```

Comment:
Potentiating doesn't throw errors, but still doesn't work using `**`.

Reply:
We removed it intentionally because it would require a much more complex RegEx. For now we reduced equation solving to the basic operators only